nixos/modules/services/networking/suricata/default.nix

```diff
@@ -0,0 +1,222 @@
+{ config
+, pkgs
+, lib
+, ...
+}: let
+  cfg = config.services.suricata;
+  pkg = cfg.package;
+  yaml = pkgs.formats.yaml { };
+  inherit (lib) mkOption types;
```

Comment:
As a matter of  preference, you should choose between either gather all `lib.XYZ` around the code here, or remove this line and use `lib.XYZ` around the code.

But this is personal TOC, I will not enforce.

Reply:
Fixed